srcpkgs/rsync/template

```diff
@@ -10,11 +10,14 @@ makedepends="zlib-devel acl-devel popt-devel xxHash-devel liblz4-devel
 short_desc="Remote fast incremental file transfer tool"
 maintainer="Lugubris <lugubris@disroot.org>"
 license="GPL-3.0-only"
-homepage="http://samba.anu.edu.au/rsync/"
+homepage="https://rsync.samba.org/"
 changelog="https://download.samba.org/pub/rsync/NEWS"
-distfiles="http://www.samba.org/ftp/rsync/src/rsync-$version.tar.gz"
+distfiles="https://www.samba.org/ftp/rsync/src/rsync-$version.tar.gz"
 checksum=becc3c504ceea499f4167a260040ccf4d9f2ef9499ad5683c179a697146ce50e
 
+# Force enable IPv6 on musl - upstream bug https://bugzilla.samba.org/show_bug.cgi?id=10715
+CFLAGS="$CFLAGS -DINET6"
```

Comment:
You can set `CFLAGS="-DINET6"` directly.

Reply:
thanks